Added freeClaim property on the networks and updated the docs template

wonderwomancode opened the pull request:
Making it clear that we have free claims and which networks they are on.

julien51 reviewed (commented):
Could you please also update the locksmith file so we use the values set here ?

julien51 reviewed (commented):
This is a good start though!

julien51 commented on packages/networks/src/networks/avalanche.ts:
```diff
@@ -41,6 +41,7 @@ export const avalanche: NetworkConfig = {
   startBlock: 17188332,
   previousDeploys: [],
   isTestNetwork: false,
+  freeClaim: 1,
```
```suggestion
  maxFreeClaimCost: 1,
```

wonderwomancode commented:
> Do you need help with the TS issue @wonderwomancode ?

Yes, now I'm not seeing why is giving me an error now that maxFreeClaimCost does not exist after I fixed the renamed ones I missed and added it to the NetworkConfig type definition.
